pyEpiabm/pyEpiabm/core/place.py

```diff
@@ -59,6 +65,14 @@ def set_max_cap(self, max_capacity: int):
         """
         self.max_capacity = max_capacity
 
+    def set_mean_cap(self, mean_capacity: float):
+        """Sets the mean capacity of a place.
+
+        :param mean_capacity: Mean capacity of place type
+        :type mean_capacity: float
+        """
+        self.mean_capacity = mean_capacity
+
     def set_infectiousness(self, infectiousness: float):
```

Comment:
I think now we don't (for mean capacity) because the relevant params are read in from file and the same across any one placetype. Good spot! 
Susceptibility may be different as might want to change that up depending on time? But you thought Covidsim susceptibility was dependent on the people not particularly the place

Reply:
see new branch